CIP-????/README.md

```diff
@@ -0,0 +1,119 @@
+---
+CIP: ?
+Title: Authenticated Web3 HTTP requests
+Category: Tools
+Status: Proposed
+Authors:
+    - Juan Salvador Magán Valero <jmaganvalero@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2022-12-27
+License: CC-BY-4.0
+---
+
+# CIP-XXXX: Authenticated Web3 HTTP requests
+
+## Abstract
+
+The cardano wallets have the ability to sign arbitrary pieces of data as we can see in the [Message signing CIP-0008](/CIP-0008/README.md). All wallets implement the method ```api.signData(addr: Address, payload: Bytes): Promise<DataSignature>``` defined in [Cardano dApp-Wallet Web Bridge CIP-0030](/CIP-0030/README.md).
+
+An important use case for web3 sites is calling HTTP endpoints using authenticated requests signed by the private keys of the users' wallets. In this way, the servers can perform actions using onchain and offchain data. 
```

Comment:
When I say the server, I mean the own dApp application server. I pursuit real decentralization, only the App server and the Blockchain is needed for this Authentication model. It can be also  used for authorization using NFTs.

I know that lately almost all Web3 Blockchain authorization and authentication models use external entities as identity authorities. But this CIP isn't about it.

I want to provide a simple and effective method for authentication without additional actors.

Reply:
@KtorZ I rewrote the abstract. Sorry for the delay. Could you please check it?

Reply:
Disambiguated as per https://github.com/cardano-foundation/CIPs/pull/442#discussion_r1182508929 and @jmagan I can see how the Abstract now includes some functional description.  It's still a bit vague and redundant, so could be improved... will leave it to @KtorZ about whether it's currently acceptable.  I would like to see one clear sentence about what the proposed component is actually doing, with less general information about Cardano and CIPs.

Reply:
I was thinking about how to improve the Abstract part but I'm clueless. I can't figure out how to write it without referencing Cardano. Shouldn't it be general and easy to understand?

Reply:
no @jmagan by all means I wasn't suggesting you not mention Cardano: just that it doesn't have to read like an article for an open audience.  If you want to set a more conversational tone, that's up to you.